run_dir/static/js/pricing_main.js

```diff
@@ -583,15 +583,15 @@ app.component('v-exchange-rates', {
           if (val === null) {
               return ""
           } else {
-              return val.toFixed(2)
+              return val
```

Comment:
I think these two should be added back as they were as well.

Reply:
Done!